site/specs-temp/numbers.json

```diff
@@ -19719,6 +17920,9 @@
         "operationId": "RetrieveSiteForThatTn",
         "summary": "Retrieve site for that tn",
         "parameters": [
+          {
+            "$ref": "#/components/parameters/AccountIdPathParam"
```

Comment:
This endpoint doesn't have account in the path, why are you adding it here?

Reply:
Thank you! rechecked all places where we don't have account id in the path and removed there.